buildSrc/src/main/kotlin/toolkit-integration-testing.gradle.kts

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Maybe this should be named `idea-integration-tests` since it's idea specific?

Reply:
We apply it to core too

Reply:
Why is the copyright 2019....